go.mod

```diff
@@ -115,14 +112,15 @@ require (
 	github.com/munnerz/goautoneg v0.0.0-20191010083416-a7dc8b61c822 // indirect
 	github.com/pkg/errors v0.9.1 // indirect
 	github.com/pmezard/go-difflib v1.0.0 // indirect
-	github.com/prometheus/client_model v0.4.0 // indirect
-	github.com/prometheus/common v0.42.0 // indirect
-	github.com/prometheus/procfs v0.9.0 // indirect
+	github.com/prometheus/client_model v0.5.0 // indirect
+	github.com/prometheus/common v0.45.0 // indirect
+	github.com/prometheus/procfs v0.12.0 // indirect
 	github.com/quic-go/qpack v0.4.0 // indirect
 	github.com/quic-go/qtls-go1-20 v0.3.1 // indirect
 	github.com/quic-go/quic-go v0.37.7 // indirect
```

Comment:
@psav  please can we bump this ? seems there's a vuln - ideally latest available (0.42.0)

Reply:
0.4.0 -> 0.42.0 ????

Reply:
or 0.4.2???

Reply:
this => https://github.com/quic-go/quic-go/releases/tag/v0.42.0

Reply:
check the Github Actions pipeline